migrations/versions/796f1055bf28_.py

```diff
@@ -0,0 +1,116 @@
+"""empty message
```

Comment:
Don't leave empty migration messages. Migrations must be traceable and understandable. Also, rename the file using some hints about the contents of the migration like "***28_optional_car_fealds.py

Reply:
Ok, sure!

Reply:
Still empty:(

Reply:
Yes, sorry, missed this :)